examples/ex5_binary_extension.py

```diff
@@ -103,13 +102,15 @@ async def main(ofrak_context: OFRAKContext, file_path: str, output_file_name: st
             attribute_filters=(ResourceAttributeValueFilter(ComplexBlock.Symbol, "main"),)
         ),
     )
+    main_cb_assembly = await main_cb.get_assembly()
     lea_instruction = await main_cb.resource.get_only_descendant_as_view(
         v_type=Instruction,
         r_filter=ResourceFilter(
             attribute_filters=(ResourceAttributeValueFilter(Instruction.Mnemonic, "lea"),)
         ),
     )
-    kitty_offset = empty_vaddr - lea_instruction.virtual_address - 7
+    ghidra_empty_vaddr = empty_vaddr + 0x100000  # Ghidra bases PIE executables at 0x100000
```

Comment:
after running `unpack_recursively`, you should be able to get the code region of the new segment (or view it as one? idk) to not hardcode the virtual address. but that's probably too verbose for here.